profile/testdata/cppbench.growth.string

```diff
@@ -3,175 +3,175 @@ Period: 1
 Samples:
 objects/count space/bytes
           1    2097152: 1 2 3 4 5 6 7 8 9 10 11 12 13 
-                bytes:[2097152]
+                bytes:[{ 2097152}]
```

Comment:
Why there are the curly braces now? Also, why there is a space after the opening brace but not before the closing one?

Reply:
The curly brackets are because the value for the numeric label is now represented as an object, rather than an int. The space is because there is an empty string for the unit field that is printed out.

Reply:
Can we make it print exactly as before when the unit is not specified?